gravitee-am-model/src/main/java/io/gravitee/am/model/permissions/Permission.java

```diff
@@ -90,8 +90,9 @@ public enum Permission {
     APPLICATION_RESOURCE(ReferenceType.ORGANIZATION, ReferenceType.ENVIRONMENT, ReferenceType.DOMAIN, ReferenceType.APPLICATION),
     APPLICATION_ANALYTICS(ReferenceType.ORGANIZATION, ReferenceType.ENVIRONMENT, ReferenceType.DOMAIN, ReferenceType.APPLICATION),
     APPLICATION_FLOW(ReferenceType.ORGANIZATION, ReferenceType.ENVIRONMENT, ReferenceType.DOMAIN, ReferenceType.APPLICATION),
+    LICENSE_NOTIFICATION(ReferenceType.ORGANIZATION, ReferenceType.ENVIRONMENT, ReferenceType.DOMAIN, ReferenceType.APPLICATION),
+    INSTALLATION(ReferenceType.ORGANIZATION);
 
-    INSTALLATION(ReferenceType.PLATFORM);
```

Comment:
**issue**: please do not change this one. PLATFORM is higher than ORGANIZATION. If we have to chance something one day, it should be adding the ORGANIZATION Reference to the INSTALLATION but PLATFORM must be preserved